docs/ru/amls.md

```diff
@@ -0,0 +1,711 @@
+# Autonomous Multirotor Landing System (AMLS) - RU
+
+![Logo](../assets/amls/logo_book.png "Logo")
+
+## Цель проекта: Автоматически посадить дрон в движении
+
+### Статья о проекте AMLS
+
+В этой статье будет описание нашего проекта, а именно, алгоритмов оптической стабилизации, удержания по GPS, следование по GPS, удержания высоты, системы захватов, крыши, измерения освещённости, скорости движения платформы. Также, после прочтения статьи станет понятно, как работает проект AMLS!
+
+### Основной репозиторий проекта на GitHub
+
+https://github.com/XxOinvizioNxX/Liberty-Way
+
+### Разработчики
+
+- Павел Нешумов: xxoinvizionxx@gmail.com
+- Андрей Кабалин: astik452@gmail.com
+- Владислав Яснецкий: vlad.yasn@gmail.com
+
+![Drone 1](../assets/amls/drone_meme.jpg "Drone 1")
+
+-----------
+
+## Оглавление
+
+- [0. Как это работает?](#0-how-it-works)
+  - [0.1. Видео про наш проект](#short-video-about-our-project-clickable)
+- [1. Удержание по GPS и полёт по точкам](#1-gps-hold-and-flight-to-waypoints-functions)
+  - [1.1. Чтение данных из UART](#11-serial-reading)
+  - [1.2. Парсинг протокола UBlox](#12-ublox-gps-parsing)
+  - [1.3. Установка текущей путевой точки](#13-set-current-waypoint)
+  - [1.4. Изменение точек (Чтобы летать по координатам)](#14-waypoint-edit-to-fly-to-waypoints)
+  - [1.5. Удержание конкретной позиции](#15-waypoint-stabilization)
+- [2. Следование за платформой](#2-gps-following)
+- [3. Компас](#3-compass)
+- [4. Удержание высоты (барометр)](#4-altitude-stabilization-barometer)
+- [5. Оптическая стабилизация](#5-optical-stabilization)
+  - [5.1. Так сложно и так важно](#51-so-difficult-and-so-important)
+  - [5.2. Первые шаги](#52-first-steps)
+  - [5.3. Инверсия](#53-inverse-approach)
+  - [5.4. Версия на Java](#54-java-edition)
+  - [5.5. Liberty-Way](#55-liberty-way)
+  - [5.6. Связь с дроном](#56-communication-with-the-drone)
+  - [5.7. Подвес для камеры](#57-camera-gimbal)
+- [6. Платформа Eitude](#6-eitude-amls-platform)
+  - [6.1. Система захватов](#61-grabbing-system)
+  - [6.2. Крыша](#62-weather-protection-system)
+  - [6.3. Спидометр](#63-platform-speedometer)
+  - [6.4. Датчики уровня освещённости](#64-platform-light-sensor)
+- [7. Заключение](#7-conclusion)
+
+-----------
+
+<a name="0-how-it-works"></a>
```

Comment:
Please, insert anchors using this syntax: ```## Title {#anchor-id}```, not using html.

Reply:
Anchors didn't work. They just displayed as code

Reply:
They will work on the main site, clover.coex.tech. See: https://clover.coex.tech/en/programming.html#positioning, https://github.com/CopterExpress/clover/blob/master/docs/en/programming.md#positioning-system-positioning.

Reply:
And this code is not working in gitbook build (for me, at least).